exporters/trace/src/main/java/com/google/cloud/operations/opentelemetry/exporter/trace/CloudTraceExporter.java

```diff
@@ -0,0 +1,51 @@
+package com.google.cloud.operations.opentelemetry.exporter.trace;
+
+import com.google.cloud.operations.opentelemetry.exporter.util.ResourceUtils;
+import com.google.cloud.trace.v2.TraceServiceClient;
+import com.google.devtools.cloudtrace.v2.AttributeValue;
+import com.google.devtools.cloudtrace.v2.ProjectName;
+import com.google.devtools.cloudtrace.v2.Span;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+public class CloudTraceExporter implements SpanExporter {
```

Comment:
`s/CloudTraceExporter/TraceExporter`? Just wanted to keep the name consistent across all the languages exporter.